kfs-core/src/main/java/edu/arizona/kfs/fp/service/impl/ProcurementCardServiceImpl.java

```diff
@@ -0,0 +1,205 @@
+        e.setTransactionLedgerEntryDescription(getUseTaxDescription(sourceDetail.getAmount(), taxRegionRate.getTaxRate()));
+        procurementCardDocument.customizeExplicitGeneralLedgerPendingEntry(sourceDetail, e);
+        procurementCardDocument.addPendingEntry(e);
+        sequenceHelper.increment();
+    
+        // handle the offset entry
+        GeneralLedgerPendingEntry offsetEntry = new GeneralLedgerPendingEntry(e);
+        generalLedgerPendingEntryService.populateOffsetGeneralLedgerPendingEntry(procurementCardDocument.getPostingYear(), e, sequenceHelper, offsetEntry);
+        procurementCardDocument.customizeOffsetGeneralLedgerPendingEntry(sourceDetail, e, offsetEntry);
+    
+        procurementCardDocument.addPendingEntry(offsetEntry);
+        sequenceHelper.increment();
+            
+        //Create the entry and offset for the Use Tax account
+        e = new GeneralLedgerPendingEntry(e);
+        e.setTransactionLedgerEntrySequenceNumber(sequenceHelper.getSequenceCounter());
+        e.setAccountNumber(taxRegion.getAccountNumber());
+        // Sub-Accounts, Sub-Object Codes or Project Codes on PCDO with Use Tax Cause GL Scrubber Errors
+        e.setSubAccountNumber(AccountingDocumentRuleBaseConstants.GENERAL_LEDGER_PENDING_ENTRY_CODE.getBlankSubAccountNumber());
+        e.setProjectCode(AccountingDocumentRuleBaseConstants.GENERAL_LEDGER_PENDING_ENTRY_CODE.getBlankProjectCode());
+        e.setChartOfAccountsCode(taxRegion.getChartOfAccountsCode());
+        e.setFinancialSubObjectCode(AccountingDocumentRuleBaseConstants.GENERAL_LEDGER_PENDING_ENTRY_CODE.getBlankFinancialSubObjectCode());
+        e.setFinancialObjectCode(taxRegion.getFinancialObjectCode());
+        e.refreshReferenceObject(KFSPropertyConstants.FINANCIAL_OBJECT);
+        e.setFinancialObjectTypeCode(e.getFinancialObject().getFinancialObjectTypeCode());
+        if (StringUtils.equals(e.getTransactionDebitCreditCode(), KFSConstants.GL_DEBIT_CODE)) {
+            e.setTransactionDebitCreditCode(KFSConstants.GL_CREDIT_CODE);
+        } else {
+            e.setTransactionDebitCreditCode(KFSConstants.GL_DEBIT_CODE);
+        }
+        e.setFinancialBalanceTypeCode(BalanceTypeService.ACTUAL_BALANCE_TYPE);
+
+        procurementCardDocument.addPendingEntry(e);
+        sequenceHelper.increment();
+        
+        // create the offset entry
+        offsetEntry = new GeneralLedgerPendingEntry(e);
+        generalLedgerPendingEntryService.populateOffsetGeneralLedgerPendingEntry(procurementCardDocument.getPostingYear(), e, sequenceHelper, offsetEntry);
+        procurementCardDocument.customizeOffsetGeneralLedgerPendingEntry(sourceDetail, e, offsetEntry);
+        procurementCardDocument.addPendingEntry(offsetEntry);
+        sequenceHelper.increment();
+        
+        return true;
+    }
+    
+    @Override
+    public boolean isUseTaxTransaction(ProcurementCardDocument procurementCardDocument) {
```

Comment:
See note above about the name of this method...
**Tech note**: Method needs java doc explanation - something similar like this: This method iterates through all the transactions entries for this pcard document and returns true ONLY if none of them satisfies one of the four conditions: AZ vendor, Sales tax already supplied, Sales Tax was edited/entered, tax exempt indicator for the current transaction is on.